app/views/subscriptions/new.html.haml

```diff
@@ -0,0 +1,37 @@
+- content_for :page_title_prefix do
+  =t('subscriptions.new')
+
+- content_for :page_description do
+  =t('subscriptions.new')
+
+.govuk-grid-row
+  .govuk-grid-column-full
+    %h1.govuk-heading-xl=t('subscriptions.new')
+.govuk-grid-row
+  .govuk-grid-column-full
+    %p.govuk-body-l=t('subscriptions.intro')
+
+    %div.govuk-inset-text
+      %ul.govuk-list
+        - @subscription.filtered_search_criteria.each_pair do |filter, value|
+          %li
+            - if filter.present?
+              %span{ class: 'govuk-!-font-weight-bold'} #{filter.humanize}:
+            =value
+
+.govuk-grid-row
+  .govuk-grid-column-one-half
+    = simple_form_for @subscription do |f|
+      = f.input :email, as: :email, input_html: { class: 'govuk-input' }, required: true
```

Comment:
Sometimes in this file we are putting spaces after `=` and sometimes not. Not sure it's worth the effort to add a rubocop for haml but I thought I'd flag in case you hadn't noticed.

Reply:
Good point. I noticed that I was doing this earlier 🙄 I'll add a card.

Reply:
https://trello.com/c/hNBToacJ/680-rubocop-investigate-add-rubocop-haml-cop

Reply:
I've also updated the file to include a space in all assignments.